providers/vmware_simulator.md

```diff
@@ -0,0 +1,98 @@
+# installing vmware simulator
+
+```bash
+cd ~/go/
+git clone https://github.com/kbrock/govmomi
+cd govmomi
+git remote add upstream  https://github.com/vmware/govmomi
+git checkout miq_counters_kb
+# client
+pushd govc ; make clean && make && make install ; popd
+# server
+pushd vcsim ; make clean && make && make install ; popd
+ln -s ~/go/bin/vcsim ~/bin # or /usr/local/bin
+ln -s ~/go/bin/govc ~/bin  # or /usr/local/bin
+```
+
+# creating simulator provider
+
+```bash
+echo "127.0.0.1 vc91" > /etc/hosts
```

Comment:
Is this necessary?  You can just use `localhost` or `127.0.0.1` when adding the provider

(I know you were trying to avoid hostname collisions with >1 vcsim but that's also more advanced than the typical UI dev will need)

Reply:
in the end of the day, vc91 (even if in `/etc/hosts`) was freaking out, so I just use localhost.